Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/status.xhtml

```diff
@@ -47,9 +47,27 @@
                 <p:tabView id="tabview" effect="fade" effectDuration="normal" dynamic="true" activeIndex="0" styleClass="tabs-override tabs-ovr-dkblue" >
                     <p:tab title="Dashboard" closable="false">
                         <div class="content-block">
+                          
+                         <p:panel rendered="#{not empty statusBean.certDTOs}"  id="alertsPanel" widgetVar="alertsPanel" style="width: 100%; margin: 0 auto;">
+                          <h2  class="subsection-header">System Alerts and Notifications </h2>
+                         <div class="table-responsive table-orgs">
+                            <p:dataTable id="certificateTable" rows="5" widgetVar="certificateTable" var="cert" value="#{statusBean.certDTOs}" scrollable="TRUE" scrollHeight="200"  styleClass="table table-striped table-status" lazy="true">
+                                <p:column>
+                                    <h:outputText rendered="#{cert.expiryColorCoding eq 'RED'}" value="Certificate with alias #{cert.alias} expired. Use Certificate Manager to Update. " style="color: red" class="text-danger" />
```

Comment:
I don't agree we should be saying "Use the certificate manager to update" on each and every entry. Can we say it once somewhere if we have any certs about to expire, or already expired?